packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -150,7 +150,7 @@ describe("MultiSelect", () => {
 
             // Assert
             expect(await screen.findByRole("button")).toHaveTextContent(
-                "1 student",
```

Comment:
Wow, that's a pretty confusing test result looking at this older code. Where did "student" even come from? Your change seems way more intuitive.

Reply:
I was wondering the same thing! Looks like "1 student" was coming from the `defaultLabels` that are passed in!
https://github.com/Khan/wonder-blocks/blob/5eedd04c9ea17bdf7141efb38ed144475da0ad4b/packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx#L28-L35